repositories.yaml

```diff
@@ -1568,3 +1568,25 @@ applications:
     channels:
       - v1_name: tiktokreporter-android
         app_id: org.mozilla.tiktokreporter
+
+  - app_name: firefox_bridge
+    canonical_app_name: Firefox Bridge
+    app_description: |
+      This is a chromium & firefox extension that allows the user to easily swap
+      between browsers on their device.
+    url: https://github.com/mozilla-extensions/firefox-bridge
+    notification_emails:
+      - ahabibi@mozilla.com
+    branch: main
+    metrics_files:
+      - "./metrics.yaml"
```

Comment:
Unusual but not wrong. Seems so far we haven't wrapped these in quotes or used a leading `./`

Reply:
I just copied the entry above. I removed the `./` and the `"`.